Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/apache/SqlConfiguration.java

```diff
@@ -29,7 +30,7 @@
     private String dbUser = "goobi";
     private String dbPassword = "CHANGEIT";
     private String dbURI = "jdbc:mysql://localhost/goobi?autoReconnect=true&amp;autoReconnectForPools=true";
-    private static final Logger logger = Logger.getLogger(MySQLHelper.class);
+    private static final Logger logger = LogManager.getLogger(MySQLHelper.class);
```

Comment:
Can you please correct the used class name from `MySQLHelper.class` to `SqlConfiguration.class` as the class name is `SqlConfiguration`?